templates/maya/render-arnold-linux/job.template.json

```diff
@@ -20,15 +20,14 @@
         },
         "mayaVersion": {
             "type": "string",
-            "defaultValue": "Maya2019",
+            "defaultValue": "Maya2019", 
```

Comment:
should this be 2020?

Reply:
recommend reverting all the changes to linux templates in this PR, and changing everything at once in a later PR as part of updating to a new linux image

Reply:
reverted this change